plugins/docker_binary_builder/app/models/binary_builder.rb

```diff
@@ -62,7 +62,7 @@ def build_file_exist?
 
   def start_build_script
     @output_stream.puts 'Now starting Build container...'
-    @container.tap(&:start).attach { |_stream, chunk| @output_stream.write chunk }
+    @container.tap(&:start).attach { |_stream, chunk| handle_output_chunk(chunk, @output_stream) }
```

Comment:
fixed

On Wed, Sep 14, 2016 at 11:04 AM, Jon Moter <notifications@github.com>
wrote:

> In plugins/docker_binary_builder/app/models/binary_builder.rb
> <https://github.com/zendesk/samson/pull/1275#discussion_r78802754>:
>
> > @@ -62,7 +62,7 @@ def build_file_exist?
> >
> >    def start_build_script
> >      @output_stream.puts 'Now starting Build container...'
> > -    @container.tap(&:start).attach { |_stream, chunk| @output_stream.write chunk }
> > +    @container.tap(&:start).attach { |_stream, chunk| handle_output_chunk(chunk, @output_stream) }
>
> Should be @ouput_stream.write_docker_chunk(chunk), right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1275/files/b834cdc37da061a8613a4888cc695aac6b9266ff#r78802754>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZxyO7m4P1F7MUdfaV7iIr_3tT5F0ks5qqDcwgaJpZM4J8_tM>
> .
>